x-pack/plugins/secops/server/graphql/network_top_n_flow/schema.gql.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const networkTopNFlowSchema = gql`
+  type NetworkTopNFlowSource {
+    ip: String
+    domain: String
+  }
+
+  type NetworkTopNFlowDestination {
+    ip: String
+    domain: String
+  }
```

Comment:
I would utilize the ECS versions rather than recreate them:

```ts
  type SourceEcsFields {
    ip: String
    port: Float
  }

  type DestinationEcsFields {
    ip: String
    port: Float
  }
```

And augment the ECS versions to have the rest of the fully flushed out object now that we need domain from the docs:
https://github.com/elastic/ecs#destination

We need the domain type within ECS anyways to visualize it now no?

That would reduce our duplicate types and fall more in line about our previous discussions about reducing our types to not having duplicates when we are technically all ECS based types.

Reply:
can not use the ECS fields parts in this query since I can not bring all the field from the query since I am using only aggregation for all the field coming back from ES query (edited) 
that’s why I created my own type for this query, if not user of the API will think that they can query other fields from the ECS but it is not the case